config/ax_cxx_compile_stdcxx.m4

```diff
@@ -33,19 +33,19 @@
 #   Copyright (c) 2014, 2015 Google Inc.; contributed by Alexey Sokolov <sokolov@google.com>
 #   Copyright (c) 2015 Paul Norman <penorman@mac.com>
 #   Copyright (c) 2015 Moritz Klammler <moritz@klammler.eu>
-#   Copyright (c) 2016 Krzesimir Nowak <qdlacz@gmail.com>
+#   Copyright (c) 2016, 2018 Krzesimir Nowak <qdlacz@gmail.com>
+#   Copyright (c) 2019 Enji Cooper <yaneurabeya@gmail.com>
```

Comment:
Why do we need to change copyright comments?

Reply:
I just took this file from GPDB7, I assume that the guys took it from the [official GNU repo](https://git.savannah.gnu.org/gitweb/?p=autoconf-archive.git;a=blob_plain;f=m4/ax_cxx_compile_stdcxx.m4;hb=44f18a92d5b6956997a9765e62711d5c70d4c2b1). I've checked the files in GNU, and they are the same as in GPDB7.

Reply:
Add to the PR description where the file was taken from

Reply:
Typos: "lebxerces", "wich". Write the branch name instead of "the base branch 6.x." for unambiguity